.circleci/config.yml

```diff
@@ -233,10 +232,10 @@ jobs:
                   corresponding_milestone_title=v$(extract_version_from_current_wd)
                   tag=$corresponding_milestone_title
                   change_notes=$(./gradlew getChangelog --no-header -q)
-                  hub release create \
-                    --attach build/distributions/git-machete-intellij-plugin-*.zip \
-                    --message "$tag"$'\n\n'"$change_notes" \
-                    $tag
+                  gh release create \
+                    build/distributions/git-machete-intellij-plugin-*.zip \
+                    --notes "$tag"$'\n\n'"$change_notes" \
+                    --notes-start-tag $tag
```

Comment:
What's the effect of using both `--notes` and `--notes-start-tag`? Looks that the latter makes sense only in conjunction with `--generate-notes`, to generate the notes out of commit messages between the `start-tag` and the `<tag>` passed as param to `gh release create`. Pls check on a separate repo

Reply:
it seems that `--notes-start-tag` is unnecessary. It does not seem to make a difference. What matters is putting the "tag" itself immediately after `create`

Reply:
I tried it here https://github.com/vzmerr/gh-test/releases